lib/matplotlib/streamplot.pyi

```diff
@@ -28,6 +28,8 @@ def streamplot(
     maxlength: float = ...,
     integration_direction: Literal["forward", "backward", "both"] = ...,
     broken_streamlines: bool = ...,
+    *,
+    n_arrows: int = 1
```

Comment:
```suggestion
    n_arrows: int = ...,
```

It is odd, but the standard in `pyi` files is to not include the actual default value, just `...` (And the extra comma is just in case we add additional kw-only args, then this line doesn't need a change)